Cargo.toml

```diff
@@ -59,7 +59,7 @@ bitstream-io = "0.8"
 cfg-if = "0.1"
 clap = { version = "2", optional = true, default-features = false }
 libc = "0.2"
-y4m = { version = "0.6", optional = true }
+y4m = { git = "https://github.com/image-rs/y4m.git", optional = true }
```

Comment:
Can we try to poke y4m-rs, people to cut a new release, not sure if it is in the schedule for them, it would be nice to have it from crates instead of git, 
Maybe @Kagami  could give us some insights over the release cycle of y4m-rs

Reply:
I've published y4m 0.7.0.